### ISSUE TITLE
mirroring of android 10 screen with audio in Linux (Ubuntu 19.10)

### ISSUE THREAD

**Issue by MrAman007** [Genymobile/scrcpy#1179]
Hi, I'm a noob so sorry if this is a repetitive question and is asked by anyone else because I didnt find my solution.
So, here it is, I want to mirror my Android 10 screen with audio to my linux pc, so that i can use obs studio to stream the screen with audio to youtube.
Currently screen mirroring is working perfectly but the audio is played only on phone. So is there any solution to get audio to linux desktop?

**Comment by rom1v** [comment]
See #14

TL;DR: for now, 2 "solutions":
 - plug a jack cable between your device and the computer;
 - use [USBaudio](https://github.com/rom1v/usbaudio) if you are lucky and your device still supports AOA audio (without bugs).

**Comment by MrAman007** [comment]
> See #14
> 
> TL;DR: for now, 2 "solutions":
> 
> * plug a jack cable between your device and the computer;
> * use [USBaudio](https://github.com/rom1v/usbaudio) if you are lucky and your device still supports AOA audio (without bugs).
Thanks, audio jack one is working USBaudio is not working unfortunately. Can i uninstall USBaudio now? Should i jest delete the file or is there any command to remove it.
